### DISCUSSION TITLE
Way to filter uncompleted kata by language?

### DISCUSSION THREAD

**Discussion by JChampt** [codewars/codewars.com#2420] [Q&A] [answered]
Hello!  

First I love this platform!  Doing katas has turned into a full on hobby for me at this point.  Thank you so much for this awesome website.  

So I have a quick question for you.  I have trained on 3 languages so far.  What I would really love to do is to sort by the following criteria: 

Sort By: Popularity
Language: (whatever I am working on at the time)
Status: Approved
Progress: Kata I have not trained on **for selected language**

The only options under Progress are; "All", "Kata I have not trained on", "Kata I have not completed", "Kata I have completed".  None of these take into account the currently selected language.  So if I want to start learning a new language I can't easily sort by only the Katas I haven't done on that specific language.  I have to sort by all, then click on each one and then click on past solutions to see if I have solved it in that particular language.  

The ability to filter/sort based on the currently selected language would be so amazing for folks that want to try out some of the top rated katas in multiple languages.  If there is a way to do that currently I have failed at finding it.  

Thanks!!

**Comment by hobovsky** [comment, marked as answer]
Please see https://github.com/codewars/codewars.com/discussions/2377

**Reply by JChampt** [reply]
Oh this is amazing.  Thank you!!!
